SimplenoteUITests/Generic.swift

```diff
@@ -45,14 +45,20 @@ class Table {
         return app.tables.element.children(matching: .cell)
     }
 
-    class func getVisibleLabelledCellsNumber() -> Int {
+    class func getVisibleLabelledCells() -> [XCUIElement] {
         // We need only the table cells that have X = 0 and a non-empty label
-        // otherwise we will include invisible elements from Sidebar pane, when Notes List is open
-        // or the elements from Notes List when Settings are open
-        // or the visible tags search suggestions
+        // Currently (besides using object dimentions) this is the way to
+        // locate note cells
         return Table.getAllCells()
             .filter { $0.frame.minX == 0.0 && $0.label.isEmpty == false }
-            .count
+    }
+
+    class func getVisibleLabelledCellsNames() -> [String] {
+        return Table.getVisibleLabelledCells().compactMap { $0.label }
+    }
+
+    class func getVisibleLabelledCellsNumber() -> Int {
```

Comment:
Nitpicks.

Maybe calling this "-count" would keep the wording consistent with what developers using Swift are used do (i.e. the `count` property).

```suggestion
    class func getVisibleLabelledCellsCount() -> Int {
```

Also, there might be only marginal benefit in having his method vs calling `.count` directly.

```swift
XCTAssertEqual(Table.getVisibleLabelledCellsCount(), 42)

// vs

XCTAssertEqual(Table.getVisibleLabelledCells.count, 42)
```

Not much difference really, the only advantage of using `.count` directly is that we'd have one less function to maintain. 🤷

Reply:
Nice! Thank you @mokagio !